elements/rh-button-group/test/rh-button-group.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { expect, html } from '@open-wc/testing';
+import { createFixture } from '@patternfly/pfe-tools/test/create-fixture.js';
+import { RhButtonGroup } from '@rhds/elements/rh-button-group/rh-button-group.js';
+
+import '@rhds/elements/rh-button/rh-button.js';
+
+describe('<rh-button-group>', function() {
+  describe('simply instantiating', function() {
+    let element: RhButtonGroup;
+    it('imperatively instantiates', function() {
+      expect(document.createElement('rh-button-group')).to.be.an.instanceof(RhButtonGroup);
+    });
+
+    it('should upgrade', async function() {
+      element = await createFixture<RhButtonGroup>(html`<rh-button-group></rh-button-group>`);
+      const klass = customElements.get('rh-button-group');
+      expect(element)
+          .to.be.an.instanceOf(klass)
+          .and
+          .to.be.an.instanceOf(RhButtonGroup);
+    });
+
+    it('renders slot correctly', async function() {
+      element = await createFixture<RhButtonGroup>(html`<rh-button-group></rh-button-group>`);
+      const slot = element.shadowRoot?.querySelector('slot');
+      expect(slot).to.exist;
+    });
+  });
+
+  describe('with buttons', function() {
+    it('should render with buttons', async function() {
+      const element = await createFixture<RhButtonGroup>(html`
+        <rh-button-group>
+          <rh-button>Save</rh-button>
+          <rh-button>Cancel</rh-button>
+        </rh-button-group>
+      `);
+      await element.updateComplete;
+
+      const buttons = element.querySelectorAll('rh-button');
+      expect(buttons).to.have.length(2);
```

Comment:
we can remove this test, it doesn't make any assertions about our element's public apis, state, or observable side effects